cmd/create/cluster/cmd.go

```diff
@@ -2209,9 +2210,12 @@ func run(cmd *cobra.Command, _ []string) {
 			r.Reporter.Errorf("%s", filterError)
 			os.Exit(1)
 		}
+
+		var excludedSubnetIdsDueToBeingPublic []string
```

Comment:
NP: Would maybe rename this var to `excludedPublicSubnets` to make the name more concise

Reply:
I see that the function you edited below has this name, I think it would be cleaner to name it this regardless, but it's a nitpick so change at your discretion

Reply:
changed it here [ed05f9e](https://github.com/openshift/rosa/pull/2832/commits/ed05f9ea971ac0d516c5f142dad1cb49cd25d7ed)